projects/archiver/src/notifications/device-notifications.ts

```diff
@@ -33,8 +62,27 @@ export const scheduleAppStoreNotification = (
         sender: 'editions-backend',
```

Comment:
What is `date` expecting here? `issueDate` is only the date and not a full timestamp.

Reply:
example that i got from Alex:

```{
"id": "9c2c1860-9ce7-41b2-9706-2f3582c99fe9",
"type": "editions",
"topic": [{"type": "editions", "name": "uk"}],
"key": "akey",
"name": "aname",
"date": "2019-09-24",
"sender": "editions-backend"
}```
just a string date